packages/playground/website/package.json

```diff
@@ -1,5 +1,5 @@
 {
 	"version": "0.0.1",
-	"type": "module",
```

Comment:
Playwright doesn't work nicely with `module`, so I switched this package to `commonjs`.

The package is private and I can't see any issues ([except for a small Cypress fix](https://github.com/WordPress/wordpress-playground/pull/1769/files#diff-51157eab23838740fda2c5de92037cf61f46af563e15e5af76489b041c723a63)). 

@adamziel @brandonpayton do you see any issues with this change?